### PR TITLE
Add 4, Link 1

### PR DESCRIPTION
**Pull request by mwchiao**
# Description
Adding 4 new icons and linking 1 app to an existing icon. QUIK is a fork of QKSMS and uses the same icon.

### Added
* Chooser (`com.uravgcode.chooser/com.uravgcode.chooser.MainActivity`)
* Claw (`dev.msfjarvis.claw.android/dev.msfjarvis.claw.android.MainActivity`)
* Daily You (`com.demizo.daily_you/com.demizo.daily_you.MainActivity`)
* Energize (`com.flasskamp.energize/com.flasskamp.energize.MainActivity`)

### Linked
* QUIK (linked `dev.octoshrimpy.quik/dev.octoshrimpy.quik.feature.main.MainActivity` to `@drawable/qksms`)

## Contributor's checklist
- [x] I followed [the Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md) (upd. Jan 2024) and will make changes if someone suggests. I will also make sure that Lawnicons builds correctly.

### CODE REVIEW

File: svgs/chooser.svg
Comment (x9136) [inline svgs/chooser.svg:1]: It is better to use 12px stroke and slightly increase the distance between the circle and the semicircle.

**Review by x9136 — CHANGES_REQUESTED** [review]

File: svgs/daily_you.svg
[the remark this answers is not in the document]
Comment (mwchiao) [reply svgs/daily_you.svg:1]: I adjusted the spacing and dropped the bottom bar a bit in my latest commit. Let me know if this is more in line with your vision.

File: svgs/chooser.svg
Comment (mwchiao) [reply svgs/chooser.svg:1]: Dropped the stroke to 12px and I'm hoping that our interpretations of "slightly" are aligned 😂.

File: svgs/daily_you.svg
[the remark this answers is not in the document]
Comment (x9136) [reply svgs/daily_you.svg:1]: It's a little high in terms of height alignment, but okay.

**Review by x9136 — APPROVED** [review]
Will you fix the conflicts?

**Pull request merged**